lib/kernel/src/application_master.erl

```diff
@@ -162,7 +162,7 @@ start_it(State, Type) ->
 init_loop(Pid, Tag, State, Type) ->
     receive
  	IoReq when element(1, IoReq) =:= io_request ->
-            relay_to_group_leader(IoReq, State),
+            _ = relay_to_group_leader(IoReq, State),
```

Comment:
Any particular reason for this '_' binding? With the notion of  "function return value(s)" I can understand, anything else apart from that? Only curious.

Reply:
It is used to suppress a dialyzer warning regarding unmatched returns.

Reply:
O!! my my dear little brain... Thanks. :smiley: